src/tests/ftest/io/ior_small.py

```diff
@@ -48,4 +48,30 @@ def test_ior_small(self):
 
         :avocado: tags=all,daosio,small,pr,hw,iorsmall
         """
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+
+        # run tests for different variants
+        self.ior_cmd.flags.update(flags[0])
+        for oclass in obj_class:
+            self.ior_cmd.daos_oclass.update(oclass)
+            for api in apis:
+                self.ior_cmd.api.update(api)
+                for test in transfer_block_size:
+                    # update transfer and block size
+                    self.ior_cmd.transfer_size.update(test[0])
+                    self.ior_cmd.block_size.update(test[1])
+                    # run ior
+                    self.run_ior_with_pool()
+
+        # Running a variant for ior fpp
+        self.ior_cmd.flags.update(flags[1])
```

Comment:
How concerned are we that the test should not produce python errors if the yaml file inputs changed?  This section for running ior fpp is highly dependent upon specific indices of various yaml inputs.  If someone want to shorten the run time of the test by commenting out a few items in the input lists this test would fail or even run with non-intended inputs (as it seems the ior fpp inputs are specific).

Reply:
I agree, but I think there are going to be such consequences of giving up with !mux for less time. If somebody wants to modify the yams they will simply need to comment these lines in the code too if they want to do some quick check.